src/borkdude/rewrite_edn/impl.cljc

```diff
@@ -76,8 +79,8 @@
                zloc zloc]
           (if (and (#{:token :map} tag) (z/rightmost? zloc))
             (-> zloc
-                (z/insert-right (node/coerce k))
-                (indent key-count first-key-loc)
+                (z/insert-right* (node/coerce k))
```

Comment:
Can you remind me what * is again?

Reply:
The `*` variant fns in the zip API insert no whitespace.